db/migrate/20201221033646_pg_search_extensions.rb

```diff
@@ -0,0 +1,11 @@
+class PgSearchExtensions < ActiveRecord::Migration[6.0]
+  def up
+    enable_extension("unaccent") unless "unaccent".in?(extensions)
```

Comment:
question: Is the `unless` condition here redundant? [The method](https://apidock.com/rails/v6.0.0/ActiveRecord/ConnectionAdapters/PostgreSQLAdapter/enable_extension) use `CREATE EXTENSION IF NOT EXISTS` which I assume would be doing the same thing?

Reply:
Yes, it's redundant, I hadn't looked at the method source code and generally try to make my migrations as idempotent as possible. I will change it, thanks!

Reply:
Updated ✅